cli/config/config.go

```diff
@@ -229,7 +229,8 @@ func initConfig() {
 func DisplayUsedConfig() {
 	// print config file
 	if !LoadedConfig && len(UserProvidedPath) > 0 {
-		log.Warn().Msg("could not load configuration file " + UserProvidedPath)
+		log.Error().Msg("could not load configuration file " + UserProvidedPath)
+		os.Exit(1)
```

Comment:
We cannot exit here. There are a couple of reasons:

- the semantics of the method name indicate it is displaying the used config. consumers of that function would not to exit the program
- I think there should be a display function, and a separate function that validates the config